src/screens/groups/components/recipients-list/recipient-info.tsx

```diff
@@ -34,7 +34,9 @@ const RecipientInfo = ({
         <Table.Cell>
           <Header as="h3">
             <Header.Content>
-              {recipient.display_name}
+              {recipient.display_name.length > 20
```

Comment:
Why do we need this?

Reply:
to fix the table row layout. as in csv file if display name is not provided then we are using wallet address in its place right, so then the table will look like this
![Screenshot from 2023-07-25 18-33-28](https://github.com/optriment/web3-employment/assets/65503300/ea8a9b32-50f7-419d-ac49-3ea69d7b6e27)

with this code - 

![Screenshot from 2023-07-25 18-36-16](https://github.com/optriment/web3-employment/assets/65503300/19a5ae2e-ccbe-43e6-94c0-38858eb3dab5)